Kconfig

```diff
@@ -24,6 +24,14 @@ config DEBUG_QUEUE_MONITOR
     help
         Enable the queue monitoring functionality.
 
+config DEBUG_PRINT_ON_UART1
+    bool "Switch DEBUG_PRINT to UART1"
+    default n
+    help
+      Switch output of DEBUG_PRINT from radio to UART1 with 115200 baudrate.
+      Note! This will interfere with several expansion decks, see:
+      https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck-pin-allocation
```

Comment:
it's a bit dangerous to put links in these places. These tend to get forgotten very easily, as we noticed lately 😄 wonder if we should make a short link to the expansiondecks instead. Or just refer them to the website and search deck pin allocation?

Reply:
With forgotten, I mean if we every change the urls of these things and such

Reply:
Agreed, pushed a commit to make it a bit safer.